about/billing.html.markerb

```diff
@@ -42,22 +48,59 @@ Stopped Machines are billed based on their root file system (rootfs) usage per s
 
 For example, a Machine described in your dashboard as having 1GB of rootfs stopped for 10 days in the entire month will cost $0.05. If you have 3 stopped Machines of 1GB rootfs each stopped for 30 days it will cost $0.45 ($0.15 x 3).
 
+---
+
 ## GPU billing
 
 GPUs are billed per second that the attached Fly Machine is running (the time they spend in the `started` state), based on the per hour cost of the GPU card. See [pricing for GPUs](/docs/about/pricing/#gpus-and-fly-machines).
 
 You're also billed for the Fly Machine separately from the GPU.
 
+---
+
 ## Volume billing
 
 Volume billing is pro-rated to the hour and we subtract the free allowances first for the Launch, Scale, and Enterprise plans. For details, see [Volume pricing](/docs/about/pricing/#persistent-storage-volumes).
 
 If you create a volume, you will be charged for it. You’re billed for volumes that aren’t attached to Machines, and for volumes that are attached to Machines in any state, including stopped Machines.
 
+---
+
+## Unified Billing
+
+Unified billing allows you to consolidate billing for multiple organizations under a single organization. This means you'll receive a single invoice for all organizations under unified billing. This feature is useful for managing billing for multiple organizations, such as when you have multiple teams or projects.
+
+**Billing Organization**: The organization that is responsible for paying for the resources used by all organizations linked to it. Any organization in good standing with an attached payment method can be a Billing Organization.
+
+**Linked Organization**: An organization that is linked to another another organization for billing purposes. It otherwise functions as a normal organization.
+
+### How Unified Billing Works
+
+- You'll receive a single invoice for all organizations under unified billing
+- Billing Organization must have a payment method on file (credit card or Stripe Link)
+- Linked Organizations share credits with their Billing Organization
+
+### How to use Unified Billing
+
+A **Linked Organization** can be created from the [new organization page](/organizations/new). When creating a new organization, you can choose an existing Organization to link it to.
+
+You can create multiple Linked Organizations for a single Billing Organization.
+
+### Some things to consider
+
+- Each organization has its own [private network](/docs/networking/private-networking/). Organizations are isolated from each other, which might
+be advantageous from a security standpoint.
+- Detaching a Linked Organization from its Billing Organization is not possible at this time. Instead we suggest creating a new organization and moving resources there.
```

Comment:
the linter really hates the word "Unlinking" so I used "Detaching" instead.

Reply:
I added "unlinking" to the linter for you if you want to use it!